### DISCUSSION TITLE
Neutral config and 'ignore' for boolean options.

### DISCUSSION THREAD

**Discussion by micheleCTDE** [uncrustify/uncrustify#3278] [Ideas]
I think it would be good if uncrustify provided a 'neutral.cfg' config file. Using this file would result in no changes whatssoever to the input file, but uncrustify would still go thorugh its formatting process.
Such file would serve as a base for people to build incremental configuration files, allowing them to evaluate one option at a time in a much easier way.

Currently it is not possible to achieve that, because booleal options only have a possible value of true/false.
In order to create a 'neutral.cfg' file, it will be necessary to add 'ignore' as a choice to all boolean options.

What do you think?

**Comment by guy-maurel** [comment]
Yes, we should try to reach it.
I am just now working on space.cpp to be sure that all the options support the value _ignore_

**Reply by micheleCTDE** [reply]
Great. I will also do some work on this in future months :+1:
